config/audit-ci.json

```diff
@@ -0,0 +1,5 @@
+{
+  "allowlist": [
+    565
```

Comment:
So this would put an exception on advisory 565 (regex vulnerability on ssri) and not advisory 565 as it applies to react-scripts. There is always the possibility that a new package can take on ssri as a dependency and use it in a way that makes 566 a real threat, but it would be over looked due to the blanket exception.

audit-ci has an alternative output where you can specify an advisory alongside a path.
https://github.com/IBM/audit-ci#testnpm-config-fileaudit-cijson